<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BitPlayer</class>
 <widget class="QWidget" name="BitPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1455</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BitPlayer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QWidget" name="bitPlayerBg" native="true">
     <property name="styleSheet">
      <string notr="true">#bitPlayerBg
{
    background-color : #FFFFFF;  /*设置背景色*/
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="head" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>68</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="headLeft" native="true">
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>24</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="logo">
              <property name="minimumSize">
               <size>
                <width>28</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#logo
{
    border-image : url(&quot;:/images/homePage/logo.png&quot;);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="headRight" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="headTittle" native="true">
              <property name="minimumSize">
               <size>
                <width>128</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>26</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>26</number>
               </property>
               <item>
                <widget class="QLabel" name="tittle">
                 <property name="minimumSize">
                  <size>
                   <width>64</width>
                   <height>14</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>64</width>
                   <height>14</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#tittle
{
    border-image : url(&quot;:/images/homePage/biteshipin.png&quot;);
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="sysBtn" native="true">
              <property name="styleSheet">
               <string notr="true">/*当光标进入到按钮时，按钮的背景颜色会发生改变*/
QPushButton:hover
{
     background-color : #E0E0E0; 
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <property name="spacing">
                <number>20</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>16</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>1154</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="minBtn">
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#minBtn
{
     border : none;
     border-image : url(&quot;:/images/homePage/suoxiao.png&quot;);
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="quitBtn">
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#quitBtn
{
     border : none;
     border-image:url(&quot;:/images/homePage/quxiao.png&quot;);
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="body" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="bodyLeft" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="btnBox" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>49</number>
               </property>
               <property name="leftMargin">
                <number>24</number>
               </property>
               <property name="topMargin">
                <number>15</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="PageSwitchButton" name="homePageBtn">
                 <property name="minimumSize">
                  <size>
                   <width>52</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>52</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="PageSwitchButton" name="myPageBtn">
                 <property name="minimumSize">
                  <size>
                   <width>52</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>52</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="PageSwitchButton" name="sysPageBtn">
                 <property name="minimumSize">
                  <size>
                   <width>52</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>52</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>538</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="bodyRight" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QStackedWidget" name="stackedWidget">
              <widget class="HomePageWidget" name="homePage"/>
              <widget class="MyselfWidget" name="myPage"/>
              <widget class="AdminWidget" name="sysPage"/>
              <widget class="UploadVideoPage" name="uploadVideoPage"/>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PageSwitchButton</class>
   <extends>QPushButton</extends>
   <header location="global">pageswitchbutton.h</header>
  </customwidget>
  <customwidget>
   <class>HomePageWidget</class>
   <extends>QWidget</extends>
   <header location="global">homepagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyselfWidget</class>
   <extends>QWidget</extends>
   <header location="global">myselfwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UploadVideoPage</class>
   <extends>QWidget</extends>
   <header location="global">uploadvideopage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AdminWidget</class>
   <extends>QWidget</extends>
   <header location="global">adminwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
